src/controllers/selectedAccount/selectedAccount.ts

```diff
@@ -69,10 +135,130 @@ export class SelectedAccountController extends EventEmitter {
     this.emitUpdate()
   }
 
+  async #updateSelectedAccountPortfolio(skipUpdate?: boolean) {
+    if (!this.#portfolio || !this.#defiPositions || !this.account) return
+
+    const now = Date.now()
+    const timeSinceLastCall = now - this.#updatePortfolioThrottle.time
+    if (timeSinceLastCall <= 150) {
+      if (!this.#updatePortfolioThrottle.throttled) {
+        this.#updatePortfolioThrottle.throttled = true
+        await wait(150 - timeSinceLastCall)
+        this.#updatePortfolioThrottle.throttled = false
+        await this.#updateSelectedAccountPortfolio()
```

Comment:
There is no use-case for this yet but why not pass skipUpdate to `await this.#updateSelectedAccountPortfolio()`. It'd get lost otherwise